tests/test_interpolation.py

```diff
@@ -93,6 +94,39 @@ def init(data):
     assert(all(np.isclose(sf.data, expected_values, rtol=1e-6)))
 
 
+@skipif_yask
```

Comment:
@tjb900 will need to use these classes with YASK. Can we actually verify that they work as expected?

Reply:
Uncommenting this, the test fails with the yask backend but that could be for a million different reasons. The [error messages](https://gist.github.com/navjotk/b2754585e734249c89d1e30a6a623406) are all cryptic so they're of no use. 

That's not the point of this PR anyway. There was a failing test submitted. This PR makes it pass.

Reply:
sure, no problem. Pity we can't use ``Precomputed*`` in YASK, though.